.github/workflows/rust.yml

```diff
@@ -31,7 +31,6 @@ jobs:
         # We don't want to commit formatting changes to main
         if: ${{ github.ref != 'refs/heads/main' }}
         with:
-          add: src
```

Comment:
I'd keep this in and whitelist the dirs we expect to change; I've seen dot files & more committed by mistake too often. Alternatively, does this support something like `**/*.rs`?

Reply:
Apparently, yes 👍